src/sensors/gpio_manager.hpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Author:
+ * Organisation: HYPED
+ * Date: 20/06/19
+ * Description:
+ *    Manages basic GPIO output for state machine and module responses for easy additions and edits
+ *
+ *    Copyright 2019 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef SENSORS_GPIO_MANAGER_HPP_
+#define SENSORS_GPIO_MANAGER_HPP_
+
+#include <cstdint>
+
+#include "sensors/manager_interface.hpp"
+
+#include "utils/system.hpp"
+#include "utils/io/gpio.hpp"
+
+namespace hyped {
+
+using utils::Logger;
+using data::Data;
+using utils::io::GPIO;
+
+namespace sensors {
+
```

Comment:
Please add docs. I have no idea what `GpioManager` is supposed to do, hence cannot review. I know lots of places lack documentation but let us try have at least some decent docs for new stuff we are adding. Even better if you can add some unit tests for the new stuff :)

Reply:
Added a [wiki](https://github.com/Hyp-ed/hyped-2020/wiki/GpioManager:-adding-digital-output-devices) for GpioManager that explains functionality if other submodules need to use similar digital output devices (ex embrakes).